### PR TITLE
Add opensuse-13 to packaging tests

### LINKED ISSUE #13507: Run Vagrant tests on OpenSUSE 13.2
Since OpenSUSE 13.2 is officially supported we should run the Vagrant tests on it

### PR DESCRIPTION
**Pull request by nik9000**
This gets opensuse-13 working with vagrant and the packaging tests. They pass
with some minor tweaks.

Closes #13507

### CODE REVIEW

**Comment by nik9000** [comment]
Ping @spinscale and @tlrx for more fun fun packaging test reviews!

File: qa/vagrant/src/test/resources/packaging/scripts/60_systemd.bats [hunk 1]
```diff
@@ -99,23 +99,27 @@ setup() {
 
 @test "[SYSTEMD] stop (running)" {
     systemctl stop elasticsearch.service
+}
 
+@test "[SYSTEMD] status (stopping)" {
     run systemctl status elasticsearch.service
-    [ "$status" -eq 3 ] || "Expected exit code 3 meaning stopped"
+    # I'm not sure why suse exits 0 here, but it does
```
Comment (tlrx) [inline qa/vagrant/src/test/resources/packaging/scripts/60_systemd.bats]: Yeah... I had to do something similar in very first version of BATS tests

**Comment by tlrx** [comment]
@nik9000 you rock! Can you please close #ht13507 in the commit message?

LGTM

**Comment by nik9000** [comment]
Force pushed with the reference to the issue. I'll merge now.

**Pull request merged**